new/LIT7107Divination.xml

```diff
@@ -150,11 +186,36 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <div type="textpart" xml:id="nagar5">
                     <label>Divinatory chapter entitled <foreign xml:lang="gez">ሀሳበ፡ ፭ ነገር፡</foreign>, or "Calculation (of things). Five"</label>
                     <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <note>Incipit taken from <ref type="mss" corresp="BAVet119"/>.</note>
                         <ab>
                             ንጉሥ፡ ብዕል፡ ሥራይ፡ ቀቲል፤ ፍቅር፤ ጥፍዓተ፤ ብእሲ፤ ሀሎ፤ ጽሑፍ፤ በማዕከለ፤ ርእሱ፤ <sic>ወሥራይ፤</sic> ይገብሩ፤ ቦቱ፤
                         </ab>
                     </div>
                 </div>
+                
+                <div type="textpart" xml:id="asrab">
+                    <label>Divinatory chapter entitled <foreign xml:lang="gez">ሐሳበ፡ አስራብ፡</foreign>, or "Calculation of the flowing waters"</label>
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <note>Incipit taken from <ref type="mss" corresp="BAVet128"/>.</note>
+                        <ab>
+                            ስመ፡ ወእመ፡ በበ<supplied reason="omitted">፡</supplied> ፲ ግድፍ፡ ፩ ንጉሥ፡ አልጱስስ፡ ወሀብቱ፡ 
+                            <sic>ሀብቱ፡</sic> ሀለወት፡ ተፈልጠ፡ እምላዕሌሁ፡ መንገለ፡
+                            ምሥራቅ፡ ኢትመጽእ፡ ፍጡነ፡
+                        </ab>
+                    </div>
+                </div>
+                
+                <div type="textpart" xml:id="fenot">
+                    <label>Divinatory chapter entitled <foreign xml:lang="gez">ሐሳበ፡ ፍኖት፡</foreign>, or "Calculation of the paths"</label>
```

Comment:
Personally, I tend to give shorter labels and avoid any markup in labels. However, I see, that this has been okay for the other textparts already existing before, so I guess, that it is acceptable also for the new ones.

Reply:
I can shorten the title deleting "Divinatory chapter entitled", here and in the other textparts. What do others think?

Reply:
I propose to shorten the label and put the rest of the information in a note. However, I think, it would be useful to do that in all the textparts to keep it consistent, if this is not too cumbersome.